src/main/drivers/pwm_output.c

```diff
@@ -130,6 +130,7 @@ static pwmOutputPort_t *pwmOutConfig(const timerHardware_t *timerHardware, uint8
 
 static void pwmWriteBrushed(uint8_t index, uint16_t value)
 {
+    value = constrain(value, 1000, 2000);
```

Comment:
I may be wrong but I think the value can exceed the 1000~2000 range with brushless ESCs, it shouldn't exceed the range only with brushed. That's why I'm putting the constrain in pwmWriteBrushed. If there's a better place please do suggest.